Misc/NEWS.d/next/Core and Builtins/2019-08-23-22-52-22.bpo-37812.nV83ZY.rst

```diff
@@ -0,0 +1,3 @@
+The ``CHECK_BINOP`` macro inside :file:`Object/longobject.c` has been
+replaced with an explicit :c:macro:`Py_RETURN_NOTIMPLEMENTED` at each call
+site, inside an ordinary ``if``.
```

Comment:
I think the Misc/NEWS entry could be a bit more succinct:

```
The ``CHECK_BINOP`` macro in :file:`Object/longobject.c` has been replaced by 
using explicit :c:macro:`Py_RETURN_NOTIMPLEMENTED`s.
```

For more details, readers can look at the changes made to ``longobject.c``.